capstone-frontend/src/App.jsx

```diff
@@ -54,10 +55,24 @@ class App extends Component {
     this.setState({ collabBookLists })
   }
 
+  fetchUserFriends = () => {
+    const userID = window.localStorage.getItem('userID');
+    fetch(`/api/user?id=${userID}`, {
+      method: 'GET',
+    }).then(resp => resp.json())
+      .then(user => user.friendIDs && Promise.all(user.friendIDs.map(friendID => {
+        return fetch(`/api/user?id=${friendID}`)
+          .then(response => response.json())
+      })))
+      .then(friends => friends && this.setState({ userFriends: friends }))
+      .catch(e => console.log(e))
```

Comment:
nit: `console.error` would probably be better here.

Reply:
Changed!